File: C/exercice3_2.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
#include <sys/types.h>
#include <sys/wait.h>
#include <time.h>
#include <errno.h>

int main(int argc, char const *argv[])
{
	if (argc != 2)
	{
		printf("argument invalide\n") ;
		return (0) ;
	}
	char *pwd = getenv("PWD") ;
	execlp("cp", "cp", "-r", pwd, argv[1], NULL) ;
	return 0;
}

File: Python/exercice3_2.py
import sys
import os

def main():
    
    if len(sys.argv) != 2:
        print("argument invalide")
        return 0
    
    pwd = os.getenv("PWD")
    
    os.execlp("cp", "cp", "-r", pwd, sys.argv[1], None)

if __name__ == "__main__":
    main()